src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -888,11 +909,14 @@ public int isNullable(int param) throws SQLServerException {
                 checkParam(param);
                 return queryMetaMap.get(param).isNullable;
             } else {
-                int nNull = (int) getParameterInfo(param).get("NULLABLE");
-                if (nNull == 1)
-                    return parameterNullable;
-                if (nNull == 0)
-                    return parameterNoNulls;
+                Map<String, Object> info = getParameterInfo(param);
+                if (null != info) {
+                    int nNull = (int) info.get("NULLABLE");
+                    if (nNull == 1)
+                        return parameterNullable;
+                    if (nNull == 0)
+                        return parameterNoNulls;
```

Comment:
Similar code compared to the switch statement at the top. Consider keeping both these consistent? E.g if we decide to use switch statements for the top implementation, also use a switch here.